VideoApp/Video-InternalTests/Mocks/MockAppSettingsStore.swift

```diff
@@ -327,72 +328,6 @@ class MockAppSettingsStore: AppSettingsStoreWriting {
         }
     }
 
-    var invokedLowRenderDimensionsSetter = false
```

Comment:
I am not opposed to adding test coverage for the new AppSettingStore properties but I would need some guidance on how to complete this.

Reply:
I don't think we need to test the settings properties in this app. The main thing is that other parts of the app that we may want to add test coverage for and depend on settings are testable. And that's what this mock is for. 

If you are curious about Swift mocks you can install this Xcode plugin and then use it to regenerate this mock: https://github.com/seanhenry/SwiftMockGeneratorForXcode

It's kind of cool. But if you have any issues just let me know and I can always run the mock generator and push to this branch. Don't want this to take your time.